modules/cudn-status-conditions.adoc

```diff
@@ -0,0 +1,84 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="cudn-status-conditions_{context}"]
+= ClusterUserDefinedNetwork status condition types
+
+The following tables explain the status conditions types returned for `ClusterUserDefinedNetwork` CRs when describing the resource. These can be used to troubleshoot your deployment.
+
+//The following table is subject to change and will be updated accordingly.
+
+.NetworkCreated condition types
+[cols="2a,2a,3a,6a",options="header"]
+|===
+
+|Condition type
+|Status
+2+|Reason and Message
+
+.3+|`NetworkCreated`
+.3+| `True`
+2+|When `True`, the following reason and message is returned:
+h|Reason
+h|Message
+
+|`NetworkAttachmentDefinitionCreated`
+|'NetworkAttachmentDefinition has been created in following namespaces: [example-namespace-1, example-namespace-2, example-namespace-3]'`
+
+.9+|`NetworkCreated`
+.9+| `False`
+2+|When `False`, one of the following messages is returned:
+h|Reason
+h|Message
+
+|`SyncError`
+|`failed to generate NetworkAttachmentDefinition`
+
+|`SyncError`
+|`failed to update NetworkAttachmentDefinition`
+
+|`SyncError`
+|`primary network already exist in <namespace_name>: <primary_network_name>`
```

Comment:
I assume this is the name of the "other" p UDN - i.e. the that already exists. 

Please confirm @ormergi .

Reply:
Correct

Status should have the following condition:
```
 conditions:
    - lastTransitionTime: "2025-02-13T12:07:43Z"
      message: 'primary network already exist in namespace "red": "test"'
      reason: SyncError
      status: "False"
      type: NetworkCreated
```

Reply:
Updated to |`primary network already exist in namespace "<namespace_name>": "<primary_network_name>"`